README.md

```diff
@@ -25,6 +25,13 @@ Make sure `~/.cargo/bin` is in your `$PATH` (or call it directly: `~/.cargo/bin/
 
 See *Requirements* Section above in case the build fails because of system dependencies.
 
+### From AUR
+you can also install from the [aur package](https://aur.archlinux.org/packages/elite-pip-macro)
+if you are using paru:
+    ``paru elite-pip-macro``
```

Comment:
```suggestion
\```
# Paru
paru -S elite-pip-macro
# Yay
yay -S elite-pip-macro
\``` 
```
(cant use 3 backticks in suggestion, but basically please use a Code Block here)